javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-player-app/script.js

```diff
@@ -0,0 +1,231 @@
+    { title: "Can't Stop Me. Can't Even Slow Me Down.", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:52" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/cant-stop-me-cant-even-slow-me-down.mp3"},
+    { title: "The Surest Way Out is Through", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "3:10" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/the-surest-way-out-is-through.mp3"},
+    { title: "Chasing That Feeling", cover: "https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/quincy-larson-album-art.jpg", artist: "Quincy Larson", duration: "2:43" , src:"https://s3.amazonaws.com/org.freecodecamp.mp3-player-project/chasing-that-feeling.mp3"},
+  ];
+
+  // song playlist
+const songListDiv = document.getElementById("playlist__songs");
+const closeBtn = document.querySelector(".playlist__close");
+
+function renderSongs(array) {
+
+// map Array Method
+const songList = songs.map((song,index) => {
+    return `
+      <li>
+        <div class="playlist__song" ${index === 0 && `aria-current="true"`}>
+            <button class="playlist__song-info">
+                <p class="playlist__song-title">${song.title}</p>
+                <p class="playlist__song-artist">${song.artist}</p>
+            </button>
+            <div class="playlist__song-duration">
+                <p>${song.duration}</p>
+            </div>
+            <button class="playlist__song-delete" aria-label="delete">
+              <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <circle id="delete-Btn" cx="8" cy="8" r="8" fill="#4d4d62"/>
+              <path fill-rule="evenodd" clip-rule="evenodd" d="M5.32587 5.18571C5.7107 4.90301 6.28333 4.94814 6.60485 5.28651L8 6.75478L9.39515 5.28651C9.71667 4.94814 10.2893 4.90301 10.6741 5.18571C11.059 5.4684 11.1103 5.97188 10.7888 6.31026L9.1832 7.99999L10.7888 9.68974C11.1103 10.0281 11.059 10.5316 10.6741 10.8143C10.2893 11.097 9.71667 11.0519 9.39515 10.7135L8 9.24521L6.60485 10.7135C6.28333 11.0519 5.7107 11.097 5.32587 10.8143C4.94102 10.5316 4.88969 10.0281 5.21121 9.68974L6.8168 7.99999L5.21122 6.31026C4.8897 5.97188 4.94102 5.4684 5.32587 5.18571Z" fill="white"/>
+              </svg>
+            </button>
+        </div>
+      </li>
+    `;
+    }).join('');
+
+    songListDiv.innerHTML = songList;
+}
+//
+renderSongs(songs);
+
+// audio API
+const audio = new Audio();
+
+// keepig track of time for each song
+let currentTime = 0;
+
+let currentIndex = 0;
+
+updatePlayingSongBkg(currentIndex);
```

Comment:
What does `Bkg` mean? If it's intended as `Background`, we should write out the full word.

Reply:
Yes, is short for background. I'll make the change